featuretools/computational_backends/utils.py

```diff
@@ -206,7 +206,7 @@ def get_client_cluster():
     return Client, LocalCluster
 
 
-def _validate_cutoff_time(cutoff_time, target_entity):
+def _validate_cutoff_time(cutoff_time, target_dataframe):
```

Comment:
Any time we grab the index or time index in this function, we need to go through the `.ww` accessor.

Reply:
updated